src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs

```diff
@@ -433,7 +429,8 @@ internal static bool TryGetEncoding(string characterSet, out Encoding encoding)
             }
             catch (ArgumentException)
             {
-                encoding = null;
+                // Use the default encoding if one wasn't provided
+                encoding = ContentHelper.GetDefaultEncoding();
```

Comment:
Please don't change functional changes in the PR.

Reply:
I will move it to another PR